test/e2e/network/kube_proxy.go

```diff
@@ -82,11 +87,10 @@ var _ = SIGDescribe("Network", func() {
 		zero := int64(0)
 
 		// Some distributions (Ubuntu 16.04 etc.) don't support the proc file.
-		_, err = e2essh.IssueSSHCommandWithResult(
-			"ls /proc/net/nf_conntrack",
-			framework.TestContext.Provider,
+		err = hostExec.IssueCommand(
+			"test -f /proc/net/nf_conntrack",
 			clientNodeInfo.node)
-		if err != nil && strings.Contains(err.Error(), "No such file or directory") {
+		if err != nil {
```

Comment:
making this error check less specific means that any infrastructure error will skip the test, rather than failing it as we want

Reply:
last commit replaces the command to test if the file exists, previously was using `ls` hence the check for the error message. I've replace `ls` by `test -f` that I think it won't need additional checks to the error message

Reply:
> I think don't need additional checks to the error message

why wouldn't we need to check the specific error? this error could be "failed to exec host command", and we would definitely not want to skip the test in that scenario

Reply:
ups, I said that  due to my ignorance, I was thinking that it receives directly the command execution output ... it seems that is better to use `ls` then. let me revert the change